module_auto_update/models/module.py

```diff
@@ -30,11 +30,15 @@ def _compute_checksum_dir(self):
         ).split(",")
 
         for r in self:
-            r.checksum_dir = dirhash(
-                get_module_path(r.name),
-                'sha1',
-                excluded_extensions=exclude,
-            )
+            try:
```

Comment:
Don't do this, but assign `dirhash = None` in import, and check here `if dirhash:`, but this should be declared as external dependency, so if the code get this, it's because you make sure the lib to be installed.